supertokens_python/recipe/dashboard/interfaces.py

```diff
@@ -66,3 +67,34 @@ def __init__(self):
         self.dashboard_get: Optional[
             Callable[[APIOptions, Dict[str, Any]], Awaitable[str]]
         ] = None
+
+
+class DashboardUsersGetResponse(APIResponse):
+    status: str = "OK"
+
+    def __init__(self, users: List[User], next_pagination_token: Optional[str]):
+        self.users = users
+        self.next_pagination_token = next_pagination_token
+
+    def to_json(self) -> Dict[str, Any]:
+        users_json = [
+            {
+                "recipeId": u.recipe_id,
+                "user": {
+                    "id": u.user_id,
+                    "email": u.email,
+                    "timeJoined": u.time_joined,
+                    "thirdParty": {}
+                    if u.third_party_info is None
+                    else u.third_party_info.__dict__,
```

Comment:
it should not be an empty object if `u.third_party_info is None`. In this case, you should not make it as a part of the JSON output